cfscrape/__init__.py

```diff
@@ -56,6 +58,21 @@
 https://github.com/Anorov/cloudflare-scrape/issues\
 """
 
+class CaptchaProvokingCiphersRemover(HTTPAdapter):
+    url = "https://"
+
+    def __init__(self, ciphers_to_remove=[], *args, **kwargs):
+        self.ciphers_to_remove = isinstance(ciphers_to_remove, (list, tuple)) and ciphers_to_remove or [ciphers_to_remove]
```

Comment:
The `cipher_to_remove` argument is meant to allow others to subclass this adapter?
If it's defined and is not an iterable, probably should throw an error.
The list of ciphers to remove is hardcoded into the library and I don't know that we want to support the option since advanced users can always mount their own custom adapter anyway.
So I'm thinking it would be better just to define the list as a constant and use in the adapter without adding arguments.

Reply:
Agree. The idea was to customize the list without touching the adapter, but in this case it doesn't really make much of a difference